nautobot/dcim/models/racks.py

```diff
@@ -310,6 +311,10 @@ class Rack(PrimaryModel, StatusModel):
         "outer_depth",
         "outer_unit",
     ]
+    dynamic_group_filter_fields = {
```

Comment:
Comment: Why is this necessary? Explain similarly to how it is done on `VirtualMachine`. And why are we still skipping fields?